scenarios/detection/11_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -41,14 +41,17 @@
   {
```

Comment:
<p>[minor] we should stick to the convention of this formatting;</p><p><code></code><code></p><p>var_assignment = a_really_long_function(</p><p>  param_1,</p><p>  param_2,</p><p>  param_3</p><p>)</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Agreed. I also ran most of the other cells through black formatter to get better formatting. That said, there was one cell in particular where the current version is much nicer to the eye.</p>